rust-runtime/aws-smithy-client/src/retry.rs

```diff
@@ -57,6 +58,7 @@ pub struct Config {
     no_retry_increment: usize,
     timeout_retry_cost: usize,
     max_attempts: u32,
+    backoff_multiplier: Duration,
```

Comment:
so my first reaction is that a duration being a multiple is a little confusing. Is this more like the initial delay?

Reply:
Your suggestion to rename this one is a good one. It is a thing that we multiply the base exponential backoff duration by but I agree calling it initial_backoff will aid users in constructing a more helpful mental model of how it works.